src/components/RandomMeme/RandomMeme.js

```diff
@@ -0,0 +1,115 @@
+import React, {Component} from "react";
+import {
+    ListItemText , Grid, Container, Box, ListItemAvatar, Avatar, ListItem
+
+} from "@material-ui/core";
+import StarIcon from '@material-ui/icons/Star';
+import StyledComponents from "./styles";
+
+class RandomMeme extends Component {
+    constructor() {
+        super();
+        this.state = {
+            randomImg: "",
+            allMemeImg: [],
+            alt: "",
+            width: "",
+            height: ""
+        };
+    }
+
+    random() {
+        let randomNumber = Math.floor(
+            Math.random() * this.state.allMemeImg.length
+        );
+        this.setState({
+            randomImg: this.state.allMemeImg[randomNumber].url,
+            alt: this.state.allMemeImg[randomNumber].name,
+            width: this.state.allMemeImg[randomNumber].width,
+            height: this.state.allMemeImg[randomNumber].height
+        });
```

Comment:
Dobry moment by wykorzystać destrukturyzacje: 
`const {length, url, name, width, height} = this.state.allMemeImg[randomNumber]`